test-driver/tsconfig.test-driver.json

```diff
@@ -0,0 +1,27 @@
+// Base Typscript Compiler Configuration
+// Targets ES6 Module system and ES6 feature set.
+{
+  "include": [
+    "src/**"
+  ],
+  "exclude": [
+    "**/*.d.ts"
+  ],
+  "compilerOptions": {
+    "target": "es6",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "esModuleInterop": true,
+    "lib": ["es6", "es2017.object"],
+    "declaration": false,
+    "inlineSources": true,
+    "sourceMap": true,
+    "strict": false,
+    "strictNullChecks": true,
+    "rootDirs": [
+      "src"
+    ],
+    "outDir": "dist",
+    "experimentalDecorators": true
+  }
+}
```

Comment:
I think the file can just be named `test-driver/tsconfig.json` since there aren't any other `tsconfig`s in the same directory.